README.md

```diff
@@ -1,108 +1,108 @@
-* [MIPS® Architecture For Programmers Volume III: The MIPS32® and microMIPS32™ Privileged Resource Architecture](http://mimiker.ii.uni.wroc.pl/documents/MD00090-2B-MIPS32PRA-AFP-06.02.pdf)
-* [MIPS32® 24KE™ Processor Core Family Software User’s Manual](http://mimiker.ii.uni.wroc.pl/documents/MD00468-2B-24KE-SUM-01.11.pdf)
-* [MIPS32® 24KEf™ Processor Core Datasheet](http://mimiker.ii.uni.wroc.pl/documents/MD00446-2B-24KEF-DTS-02.00.pdf)
-* [Programming the MIPS32® 24KE™ Core Family](http://mimiker.ii.uni.wroc.pl/documents/MD00458-2B-24KEPRG-PRG-04.63.pdf)
-* [MIPS® YAMON™ User’s Manual](http://mimiker.ii.uni.wroc.pl/documents/MD00008-2B-YAMON-USM-02.19.pdf)
-* [MIPS® YAMON™ Reference Manual](http://mimiker.ii.uni.wroc.pl/documents/MD00009-2B-YAMON-RFM-02.20.pdf)
-* [MIPS ABI Project](https://dmz-portal.mips.com/wiki/MIPS_ABI_Project)
-
-Hardware documentation:
-* [MIPS® Malta™-R Development Platform User’s Manual](http://mimiker.ii.uni.wroc.pl/documents/MD00627-2B-MALTA_R-USM-01.01.pdf)
-* [Galileo GT–64120 System Controller](http://doc.chipfind.ru/pdf/marvell/gt64120.pdf)
-* [Intel® 82371AB PCI-TO-ISA/IDE XCELERATOR (PIIX4)](http://www.intel.com/assets/pdf/datasheet/290562.pdf)
-* [Am79C973: Single-Chip 10/100 Mbps PCI Ethernet Controller with Integrated PHY](http://pdf.datasheetcatalog.com/datasheet/AdvancedMicroDevices/mXwquw.pdf)
-* [FDC37M81x: PC98/99 Compliant Enhanced Super I/O Controller with Keyboard/Mouse Wake-Up](http://www.alldatasheet.com/datasheet-pdf/pdf/119979/SMSC/FDC37M817.html)
+# Mimiker: Unix-like system for education and research purposes
+
+Mimiker's main goal is to deliver minimal Unix-like operating system, i.e.
+the kernel and a set of userspace programs.
+
+Kernel design is heavily inspired by FreeBSD & NetBSD systems with some ideas
+taken from Linux, Plan9 and other OSes. We spend a lot of time reading source
+code of open-source operating systems. We carefully choose their best design
+decisions, ideas, algorithms, APIs, practices and so on, distill them to bare
+minimum and reimplement them or adapt to Mimiker code base. We hope not to
+repeat their mistakes and move away from legacy and non-perfect solutions.
+
+Mimiker project gathers like minded people who value minimalism, simplicity and
+readability of code. We strive for the lowest possible complexity of solutions.
+We love to throw away code that isn't terribly useful or handles rare edge
+cases. We know value of debuggability and we don't hesitate to spend time
+writing tools that help to improve it.
+
+Though userspace programs are part of Mimiker project, they've got simply ported
+from NetBSD or [suckless][1] project. We focus on kernel development, since we
+find it more interesting. We don't want to invest too much time into the device
+drivers, so we keep a list of target platforms small.
+
+If you'd like to get involved in the project please read our [Wiki][2] to find
+out more!
+
+## Where we are?
```

Comment:
Shouldn't it be "Where are we?" (or imho better: "Where we are", without the question mark).